latest/docs-ref-conceptual/install-azure-cli.md

```diff
@@ -24,46 +24,34 @@ For information about the latest release, see the [release notes](release-notes-
 > [!NOTE]
 > If you need the previous version of the Azure CLI, here's how to [install Azure CLI 1.0](/azure/cli-install-nodejs).
 
-## macOS
+## <a name="macOS"/>Install on macOS
 
-> [!WARNING]
-> The Homebrew formula for the Azure CLI, `azure-cli`, is currently out of date and will install a previous version.
-
-1. Install Azure CLI 2.0 with one `curl` command.
+1. Install Azure CLI 2.0 with `curl`.
 
    ```bash
    curl -L https://aka.ms/InstallAzureCli | bash
    ```
 
-2. You may have to restart your command shell for some changes to take effect.
+2. You may have to restart your shell for some changes to take effect.
 
    ```bash
    exec -l $SHELL
    ```
    
 3. Run the CLI from the command prompt with the `az` command.
 
-> [!Note]
-> When you install with InstallAzureCli, [`az component update`](/cli/azure/component#update) isn't supported.
-> To update to the latest CLI, run `curl -L https://aka.ms/InstallAzureCli | bash` again.
-> 
-> To uninstall, see the [manual uninstall instructions](#uninstall).
-
-## Windows
+> [!WARNING]
+> The Homebrew formula for the Azure CLI, `azure-cli`, is currently out of date and will install a previous version.
```

Comment:
Providing explicit instructions on how to install with homebrew _while at the same time saying __don't do this___ is kind of ridiculous, especially since users aren't really going to have a good way to know when `azure-cli` is updated unless they're polling `brew info` regularly or grepping `brew update` for the package name. The plan is to update the instructions once the brew keg is correct.

As a result the wording is trickier and may need to be stronger and more specific:

> Avoid installing the `azure-cli` Homebrew formula. It is out of date and will install Azure CLI 1.x. This page will be updated with new installation instructions when the Homebrew formula is correct, and you can check `brew info azure-cli` to see what the current available version is.